mkdocs.yml

```diff
@@ -310,3 +308,43 @@ plugins:
         redirect_maps:
             'concepts/manifest.md': 'concepts/environments.md#manifesttoml'
             'install-flox.md': 'install-flox/install.md'
+            'reference/command-reference/flox-activate.md': 'man/flox-activate.md'
```

Comment:
Did you check whether there was a glob syntax you could use instead of listing these individually?